src/vm/threads.cpp

```diff
@@ -12170,7 +12171,7 @@ HRESULT Thread::YieldTask()
             break;
         }
 
-#ifdef FEATURE_HIJACK
+#if defined(FEATURE_HIJACK) && !defined(PLATFORM_UNIX)
```

Comment:
It may be better to put the whole YieldTask method under #ifdef FEATURE_INCLUDE_ALL_INTERFACES.